applications/FSIapplication/tests/convergence_accelerator_spring_MPI_test.py

```diff
@@ -82,8 +82,6 @@ def ReadModelPart(self,filename):
         model_part.AddNodalSolutionStepVariable(KratosMultiphysics.FSI_INTERFACE_RESIDUAL)
         mpi.world.barrier()
 
-        KratosMetis.SetMPICommunicatorProcess(model_part).Execute()
```

Comment:
@rubenzorrilla you later use the `ParallelFillCommunicator` which also sets the MPI-comm, hence I think you can safely remove this